bciers/apps/reporting/src/app/components/routes/operations/Page.tsx

```diff
@@ -1,23 +1,32 @@
 import { getReportingYear } from "@reporting/src/app/utils/getReportingYear";
 import Operations from "../../operations/Operations";
 import { OperationsSearchParams } from "../../operations/types";
+import dayjs from "dayjs";
 
+const formatDate = (dateString: string | number | Date) => {
+  return dayjs(dateString).format("MMM DD,YYYY");
+};
 export default async function OperationsPage({
   searchParams,
 }: {
   searchParams: OperationsSearchParams;
 }) {
   const reportingYearObj = await getReportingYear();
 
+  const reportDueDate = reportingYearObj?.report_due_date
```

Comment:
`reportingYearObj.report_due_date` is bound to exist here, there is no need to handle the falsy case. This would swallow an error if there is one with the fetch